### ISSUE TITLE
problems with contenteditable

### ISSUE THREAD

**Issue by eostrom** [teampoltergeist/poltergeist#432]
First off, thanks to @pedrocarrico and @route for putting #416 into the 1.5 release. Now, the trouble starts....

I'm having trouble using `set` on multiple `contenteditable` nodes. I don't have a minimal test case yet, but to give you an idea: I have a series of `td` elements that currently contain "0", and I want to set some of them to different values:

```
["1", "5.8", no change, "2", "", no change]
```

So I have a loop that finds each `td` and calls `set` on it. After each change, I log the position changed, the new value, and an array of `tds.map(&:text)`. When I run it, I get:

```
BEFORE ["0", "0", "0", "0", "0", "0"]
CHANGED 1 "1" ["01", "0", "0", "0", "0", "0"]
CHANGED 2 "5.8" ["0", "05.8", "0", "0", "0", "0"]
CHANGED 4 "2" ["0", "05.", "0", "02", "0", "0"]
CHANGED 5 "" ["0", "05.", "0", "0", "0", "0"]
AFTER ["0", "05.", "0", "0", "0", "0"]
```

When I `set` the first `td` to "1", it appended it to "0" instead of replacing it. Then, when I `set` the second `td` to "5.8", it deleted the "1" from the first `td` and then appended "5.8" to second `td`. That pattern continues: delete one character from the previous `td`, then append the new string to the current one.

Long story short, it looks like the "delete text and send keys" approach is deleting the wrong text. I thought it might be a focus problem, so I started calling `td.click` before each `td.set`, and it works: each `set` replaces rather than appending, and affects only its own element.

I will try to produce a test case and pull request, but I thought I'd get the conversation started first....
